ci-operator/step-registry/sandboxed-containers-operator/gather-must-gather/sandboxed-containers-operator-gather-must-gather-ref.yaml

```diff
@@ -0,0 +1,30 @@
+ref:
+  as: sandboxed-containers-operator-gather-must-gather
+  best_effort: true
+  optional_on_success: false
+  from: cli
+  commands: sandboxed-containers-operator-gather-must-gather-commands.sh
+  env:
+  - name: ENABLE_MUST_GATHER
+    default: "true"
+    documentation: |-
+      Whether to run the must-gather step. Set to "false" to skip must-gather collection.
+  - name: MUST_GATHER_ON_FAILURE_ONLY
+    default: "false"
+    documentation: |-
+      When set to "true", only run must-gather if tests failed. When "false", always run must-gather (if enabled).
+  - name: MUST_GATHER_TIMEOUT
+    default: 35m
+    documentation: A timeout for oc adm must-gather command.
+  - name: MUST_GATHER_IMAGE
+    default: "registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel9:latest"
+    documentation: |-
+      The must-gather image to use for collecting sandboxed containers operator diagnostics.
+  resources:
+    requests:
+      cpu: 300m
+      memory: 300Mi
+  timeout: 2h
```

Comment:
mustgather timeout is 35m, wouldn't 45m do here?

Reply:
Done